extensions/amp-story/1.0/amp-story.js

```diff
@@ -424,6 +428,14 @@ export class AmpStory extends AMP.BaseElement {
         }
       });
     }
+
+    if (this.element.hasAttribute('dynamic-live-list')) {
```

Comment:
Sorry, what's the proposed API here?

`<amp-story dynamic-live-list="myLiveList">`?  Is this publisher-specified?  Can't we just allocate an ID ourselves so that publishers don't have to?